contracts/ModuleFactory.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "./modules/RevokeModule.sol";
+import "./modules/AaveWithdrawModule.sol";
+
+import "interfaces/gnosis/IGnosisSafe.sol";
```

Comment:
this already exists as `interfaces/ISafe.sol`

Reply:
did bit cleanup in 8e3b3409377ffa9b9ddb518c903187b52259d285